modules/nf-core/kmcp/index/main.nf

```diff
@@ -0,0 +1,37 @@
+process KMCP_INDEX {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::kmcp=0.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
+        'quay.io/biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(outdir)
```

Comment:
`outdir` might be a bit confusing to use as input name

Reply:
Changed to `compute_output` since the input of `kmcp/index` is the output of `kmcp/compute`.

Reply:
I would refrain from using anything with `out` or `output` as input to prevent confusion. Can't you use `compute_dir` or something instead?

Reply:
Ok, I see!